### ISSUE TITLE
COPS name is too generic and hard to search on

### ISSUE THREAD

**Issue by obscuretech** [seblucas/cops#318]
The project name "COPS" is nearly impossible to search outside GitHub because the name is a common word in English that produces many, many unrelated links when typed in a search engine. It's also not precise: COPDS would be more accurate given its description as 'Calibre OPDS'.

**Comment by seblucas** [comment]
Of course you're right, that's certainly why my main job is Software engineer and not Marketing director ;)

When I created this project, I found that "COPS" sounded funny especially in relation to Calibre. COPS offer OPDS and HTML gui, I've not made any statistics but I think the HTML catalog is mostly used.

For now I'm not a big fan of renaming it only because of search engines, maybe there are other arguments ?

**Comment by Ecksters** [comment]
I agree that the name is generic, however, even the first page of ODPS is taken by a whole bunch of Ohio departments, so unfortunately it's simply not a popular enough concept to even have its main term beat out government agencies.

While searching for COPDS would certainly mean you could find the program easier if you know the name, searching for Calibre OPDS would get that as well. However, I agree with you if one could go back in time, a more unique name may have been a good idea, perhaps even just a cute name instead of an abbreviation.

As it stands, I feel like if anything subtitles should be changed to catch the "Home Ebook Server" folks who are likely trying to find this kind of software.

**Comment by horus68** [comment]
Agree on the difficulty when searching. Recommending the software to someone on a conversation and asking them to google it would be a bad thing... so I just ask the email to send them a link!
"Cops" in english is used for "Police" so its a difficult name for a software when web searching!
Re-branding is always difficult and not easy to please to all. Specially to the developer. 
But a longer name would be better to start with, acting as keywords.
Making new long name visible on every software page near COPS would make it easier for google to display it on listings so people knew they are not going into a police page! 

And... an image/logo too is required (there's a PR for that!) 👍 

On name ideas: 
- "Calibre OPDS" would be a generic therm as there are others (even if COPS is be best!). 
- Avoid OPDS near COPS or you will end up with too meany abbreviations.
- Acronyms are usually good.
- There must be something related to ebook on the name!

for now make it simple: "COPS ebookserver"
(note there is no space on ebookserver to look like a branding not single words)

1- COPS ebookserver = 11 500 results (mostly relevant)
https://www.google.pt/search?q=COPS+ebookserver

2- COPSebookserver = 6 results (all relevant!)
https://www.google.pt/search?q=COPSebookserver

3- COPSebook = 3 550 results (many not relevant)
https://www.google.pt/search?q=COPSebook

Or a different play around... keeping COPS but in a single work
4- EbookCOPServer
